operator/gitops/argocd/pipeline-service/tekton-results/watcher-config.yaml

```diff
@@ -21,5 +21,12 @@ spec:
               "-completed_run_grace_period",
               "10m",
             ]
+          resources:
+            requests:
+              cpu: 5m
+              memory: 64Mi
+            limits:
+              cpu: 250m
+              memory: 2Gi
```

Comment:
@sayan-biswas I prod rh01 I see we can consume up to 5.27Gi. Do we want to bump this value to 6Gi?

Reply:
fyi folks I just posted a comment in https://issues.redhat.com/browse/PLNSRVCE-867 with some references to the profiling/pprof work the performance wg did with the tekton pipelines controller earlier this year.

Reply:
Based on @sayan-biswas comment, we can keep the limit to 2Gi so I am resolving this thread